src/Platforms/Echo.Platforms.AsmResolver/Emulation/Heap/ObjectMapMemory.cs

```diff
@@ -125,8 +125,8 @@ public void Unmap(object value)
         /// </summary>
         public void Clear()
         {
-            foreach (long address in _objectsByAddress.Keys)
-                _backingBuffer.Unmap(address);
+            foreach (var map in _objectsByAddress.Values)
```

Comment:
We should probably rename `_objectsByAddress` to `_objectsByOffset` so that in the future this bug does not creep back in.